### PR TITLE
chore(tests): allow unknown args in e2e

### PR DESCRIPTION
**Pull request by jhoward1994**
### What does it do?

Specify debug as a yarg option

### Why is it needed?

So we can run things like 
`yarn test:e2e --domains content-manager -- tests/e2e/tests/content-manager/cloning.spec.ts --debug`
&
`yarn test:e2e --domains content-manager -- tests/e2e/tests/content-manager/cloning.spec.ts`

### CODE REVIEW

File: tests/scripts/run-e2e-tests.js [hunk 1]
```diff
@@ -191,6 +198,8 @@ yargs
           return acc;
         }, []);
 
+        const debugFlag = debug ? ['--debug'] : [];
```
Comment (innerdvations) [inline tests/scripts/run-e2e-tests.js:201]:
```suggestion
        const optionFlags = [];
        if(debug) {
          optionFlags.push('--debug');
        }
```
Something like that would be more future-proof if we have any other options to add, but I'm also fine with it how you have it

**Review by innerdvations — APPROVED** [review]
LGTM!

**Review by innerdvations — APPROVED** [review]

**Comment by jhoward1994** [comment]
hey @joshuaellis would you be able to take a look at this PR if you have as sec please? I can't see why these changes would cause the e2e tests to fail in this way, the same specs are passing locally for me but the CI has been failing in the same way each time :(

**Comment by innerdvations** [comment]
Just had a thought. Instead of adding the debug option, why don't we just change the package.json script to use `-- --debug` to pass the arg instead of manually adding it?

**Comment by innerdvations** [comment]
Or even simpler solution for now, I'm also fine with flipping it back to `true` and updating the documentation to specify that if you want to run individual files, you must not do it after the --domains arg, ie, you have to do `yarn test:e2e:debug --setup -c=1 sometest.spec.ts --domains=admin` but that `yarn test:e2e:debug --setup -c=1 --domains=admin sometest.spec.ts` will fail.

**Comment by jhoward1994** [comment]
Do you mean to make this the command in package.json `"test:e2e:debug": "node tests/scripts/run-e2e-tests.js -- --debug"`

I feel like thats worse as you then can't do something like `yarn test:e2e:debug --concurrency=1 --domains=content-manager`

We could remove the `:debug` entry in package.json completely and document to add `-- --debug` to run in debug mode?

I'm not sure I understood your second comment, have I missed something?

**Review by innerdvations — APPROVED** [review]
Thanks! I like this way because it keeps our script clearly separate from playwright options. I think we can also revisit later and find a clean way to add back the test:e2e:debug if we want.

I'll take care of making similar changes on CLI tests in the next few weeks as I start to merge them.

**Pull request merged**